float32 leftUSDist
float32 frontUSDist
float32 rightUSDist
